datadog/detect.go

```diff
@@ -12,22 +12,27 @@ import (
 
 	"github.com/buildpacks/libcnb"
 	"github.com/paketo-buildpacks/libpak"
+	"github.com/paketo-buildpacks/libpak/bard"
 )
 
-type Detect struct{}
+type Detect struct{
+	Logger bard.Logger
+}
 
 func (d Detect) Detect(context libcnb.DetectContext) (libcnb.DetectResult, error) {
-	cr, err := libpak.NewConfigurationResolver(context.Buildpack, nil)
+	cr, err := libpak.NewConfigurationResolver(context.Buildpack, &d.Logger)
 	if err != nil {
 		return libcnb.DetectResult{}, fmt.Errorf("unable to create configuration resolver\n%w", err)
 	}
 
 	if !cr.ResolveBool("BP_DATADOG_ENABLED") {
+		d.Logger.Info("SKIPPED: variable 'BP_DATADOG_ENABLED' not set to true")
 		return libcnb.DetectResult{Pass: false}, nil
 	}
 
 	// If both BP_DATADOG_ENABLED and BP_NATIVE_IMAGE are enabled, don't require jvm-application plan and prepare for native-image only
 	if cr.ResolveBool("BP_NATIVE_IMAGE") {
+		d.Logger.Info("PASSED: native-image was enabled via BP_NATIVE_IMAGE")
```

Comment:
well, it's not an error in that case, but in the case the user uses `verbose` then I guess it could help

Reply:
Exactly 👍